### PR TITLE
Add docs for updating reva and releasing ocis-reva

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

**Comment by PVince81** [comment]
Phoenix tests failure unrelated :facepalm:

File: docs/releasing.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{{< / highlight >}}
+6. Create a pull request to the master branch.
+
+## Release
+1. After the preparation branch has been merged update your local master.
+{{< highlight txt >}}
+git checkout master
+git pull
+{{< / highlight >}}
+2. Create a new tag (preferably signed).
```
Comment (PVince81) [inline docs/releasing.md:38]: @micbar so I guess we should add "wait for CI" before tagging ?

File: docs/releasing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{{< highlight txt >}}
+git tag -s vx.x.x -m "release vx.x.x"
+git push --tags
```
Comment (micbar) [inline docs/releasing.md:41]: I always do a `git push origin tagname`
This is more explicit.

File: docs/releasing.md (hunk 1 above)
Comment (PVince81) [reply docs/releasing.md:38]: actually need to wait for the CI job that updates the changelog, and tag on that commit

**Comment by PVince81** [comment]
@micbar I've adjusted the PR, see last commit

File: docs/releasing.md [hunk 3]
```diff
@@ -25,22 +25,23 @@ mv changelog/unreleased/* changelog/x.x.x_yyyy-MM-dd/
...
 git add --all
 git commit -m "prepare release x.x.x"
-git push
+git push origin release-x.x.x
 {{< / highlight >}}
 6. Create a pull request to the master branch.
 
 ## Release
 1. After the preparation branch has been merged update your local master.
+2. [Wait for CI])https://cloud.drone.io/owncloud/ocis-reva) to generate a commit for the changelog update
```
Comment (micbar) [inline docs/releasing.md:34]: typo, wrong bracket

Comment (PVince81) [reply docs/releasing.md]: fixed

**Pull request merged**